src/styles/theme.scss

```diff
@@ -42,32 +35,49 @@ $white: #f5f5f5;
 $grey-light: #eee;
 $grey-light-purple: #e4e1e7;
 $lime-green: #07c103;
+*/
+
+// Generic Variables
+$yellow: #FEDD00;
+$yellow-lightest: #FFEF82;
+$blue: #00418F;
+$blue-lightest: #C8DDF6;
+$blue-light: #407EC9;
+$blue-dark: #47195b;
+$black: #3C3C3B;
+$white: #fbfbfb;
+$grey-light: #F9F9F9;
+$grey: #F5F5F5;
+
 
 // Utility variables
 $white-pure: #ffffff;
 $black-pure: #000000;
 $shadow: rgba(89, 97, 116, 0.4);
 
 // Theme colours
-$primary-colour: $blue-darkest;
-$secondary-colour: $blue-light;
+$primary-colour: $yellow;
+$secondary-colour: $blue;
 $text-colour: $black;
-$interaction-colour: $purple;
-$focus-colour: $lime-green;
+$text-colour-alt: $grey;
+$interaction-colour: $yellow;
+$focus-colour: $blue-light;
+$hover-colour: $yellow-lightest;
+$background-highlight: $blue-light;
 
 /* ------------------------------
 
    Defaults
 
 ------------------------------ */
 
-*:focus {
-  outline-color: $focus-colour;
+:focus {
```

Comment:
This shouldn't change anything, but I would avoid unnecessary deviations from the core sheet.